src/main/java/com/jwplayer/southpaw/Southpaw.java

```diff
@@ -267,39 +267,43 @@ protected void build(int runTimeS) {
                     while (records.hasNext()) {
                         ConsumerRecord<BaseRecord, BaseRecord> newRecord = records.next();
                         ByteArray primaryKey = newRecord.key().toByteArray();
+
+                        logger.info("---------------------------------");
```

Comment:
should probably remove this in the final merge

Reply:
Done